tools/updateRollup/updateRollup.ts

```diff
@@ -99,17 +108,124 @@ async function main() {
         throw new Error(`Address ${deployer.address} does not have the UPDATE_ROLLUP_ROLE role`);
     }
 
-    // Add a new rollup type with timelock
-    console.log(await (await rollupManagerContract.updateRollup(rollupAddress, newRollupTypeID, upgradeData)).wait());
+    const outputsJson = [];
+
+    for(let i = 0; i < updateRollupsParameters.rollups.length; i++) {
+        const outputJson = {} as any;
+        const updateRollupParameters = updateRollupsParameters.rollups[i];
+
+        /*
+         * Check parameters
+         * Check that every necessary parameter is fullfilled
+         */
+        const mandatoryParametersRollup = [
+            "rollupAddress",
+            "newRollupTypeID",
+            "upgradeData",
+        ];
+
+        for (const parameterName of mandatoryParametersRollup) {
+            if (updateRollupParameters[parameterName] === undefined || updateRollupParameters[parameterName] === "") {
+                throw new Error(`Missing rollup[${i}] parameter: ${parameterName}`);
+            }
+        }
 
-    outputJson.networkName = network.name;
-    outputJson.polygonRollupManagerAddress = polygonRollupManagerAddress;
-    outputJson.rollupAddress = rollupAddress;
-    outputJson.newRollupTypeID = newRollupTypeID;
-    outputJson.upgradeData = upgradeData;
+        const {rollupAddress, newRollupTypeID, upgradeData} = updateRollupParameters;
+
+        outputJson.networkName = network.name;
+        outputJson.polygonRollupManagerAddress = polygonRollupManagerAddress;
+        outputJson.rollupAddress = rollupAddress;
+        outputJson.newRollupTypeID = newRollupTypeID;
+        outputJson.upgradeData = upgradeData;
+
+        if(updateRollupsParameters.type === transactionTypes.EOA) {
+            console.log("Updating rollup...")
```

Comment:
Add `rollupAddress` that is being updated.
I think it will help to track the progress of the script.
Same for multisig/timelock